datalad/utils.py

```diff
@@ -369,14 +369,36 @@ def _is_stream_tty(stream: Optional[IO]) -> bool:
 
 
 def is_interactive() -> bool:
-    """Return True if all in/outs are open and tty.
+    from datalad import cfg
+    return cfg.obtain("datalad.ui.interactive")
```

Comment:
what if config is within local git repo config?

shouldn't we overload that DATALAD_UI_INTERACTIVE here? and if we are to decide for "auto" to be default -- then here call is_interactive_failsafe explicitly?